### PR TITLE
ufm-metamask prod envs and refactors

### PR DESCRIPTION
**Pull request by Spacesai1or**
- Adds ENVs and refactors read/write metrics to make deployment to Kubernetes work
- Replace use of OP Goerli with OP Sepolia
- Removes actual Github Workflow file (made it into an example file since we're now using K8s)

### CODE REVIEW

**Comment by Spacesai1or** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #7576** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7576" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7576?utm_source=stack-comment).

File: ufm-test-services/.env.example [hunk 1]
```diff
@@ -5,14 +5,24 @@ CI=false
...
-# If true, Metamask Test Service will install Xvfb inside it's container and used that for Playwright tests.
-# If false, Metamask you will need to specify METAMASK_DISPLAY and METAMASK_DISPLAY_VOLUME so Playwright can connect to a display
-METAMASK_RUN_HEADLESS=true
+METRICS_WRITE_URL="http://pushgateway:9091"
+# Dictates how the request body is structured when pushing metrics. Should be either "grafana" or "prometheus-pushgateway"
+METRICS_WRITE_TOOL="prometheus-pushgateway"
+# This is the source the pushed metric will be labeled as originting from. May not need this value
+METRIC_WRITE_SOURCE=""
+# The needed credentials to access METRICS_READ_URL. Will be sent as: Authorization: Bearer username:password
+METRICS_WRITE_USERNAME=""
```
Comment (hamdiallam) [inline ufm-test-services/.env.example:20]: q: do the write credentials also work as read credentials? Wondering if we could consolidate the two

File: ufm-test-services/.env.example [hunk 2]
```diff
@@ -24,13 +34,13 @@ METAMASK_DISPLAY_VOLUME=/tmp/.X11-unix:/tmp/.X11-unix
 METAMASK_SECRET_WORDS_OR_PRIVATEKEY="test test test test test test test test test test test junk"
 
 # The initial network Metamask will be initialized with, Test Service will override with OP Goerli
```
Comment (hamdiallam) [inline ufm-test-services/.env.example]:
```suggestion
# The initial network Metamask will be initialized with, Test Service will override with OP Sepolia
```

**Review by hamdiallam — APPROVED** [review]
LGTM!

General Q: Is the switch to sepolia just for the example/unit tests? Will the test service be spun up against both georli and sepolia?

**Review by roninjin10 — COMMENTED** [review]
Random thought, do you think we could add observability into both what the current version of metamask is and what version is being tested here?

File: ufm-test-services/.env.example (hunk 1 above)
Comment (Spacesai1or) [reply ufm-test-services/.env.example:20]: No they won't. There might be instances (for other OP chains) where they do, but for our use case they're different

**Comment by Spacesai1or** [comment]
> Random thought, do you think we could add observability into both what the current version of metamask is and what version is being tested here?

@roninjin10 From my understanding, not using Synpress. We're beholden to their latest implemented version of Metamask

**Comment by Spacesai1or** [comment]
> LGTM!
> 
> General Q: Is the switch to sepolia just for the example/unit tests? Will the test service be spun up against both georli and sepolia?

We discussed on our 1:1, but supposedly OP Goerli is being deprecated soon™️

**Review by trianglesphere — APPROVED** [review]

**Review by trianglesphere — APPROVED** [review]

**Review by felipe-op — APPROVED** [review]

**Pull request merged**